RESOURCES.md

```diff
@@ -37,4 +37,14 @@
 [React - The Complete Guide](https://www.udemy.com/course/react-the-complete-guide-incl-redux/)   
 [React JS Crash Course 2021 - Traversy Media](https://youtu.be/w7ejDZ8SWv8)   
 [React JS Course for Beginners - 2021 Tutorial](https://youtu.be/nTeuhbP7wdE)   
-[Epic React - Kent Dodds ($$$)](https://epicreact.dev/)   
\ No newline at end of file
+[Epic React - Kent Dodds ($$$)](https://epicreact.dev/)
+
+
+## Backend Resources
+
+[JSON Formatter](goo.gl/ZDLWY0)
+
+
+## Solidity Tutorial
+
+[Solidity for Beginners - Crypto Zombies Course](https://cryptozombies.io/)   
```

Comment:
While this looks like a great resource (I've never seen it before!!), we're keeping this file with resources just related to the scrollodex project. Can you remove? Thank you!!